core/jazz_splunk-kinesis-log-streamer/index.js

```diff
@@ -36,8 +36,9 @@ function handler(event, context, callback) {
   logger.debug('Received event:' + JSON.stringify(event));
   let errorHandler = errorHandlerModule();
   let config = configData.getConfig(event, context);
+  let isSplunkEnabled = (global_config.ENABLE_SPLUNK).toLowerCase() == 'true' ? true : false;
```

Comment:
Check if global_config.ENABLE_SPLUNK exists & is a string before converting it into lowercase. Return 5XX (since it's a service configuration error) if the value is invalid/empty.

Reply:
Updated.